optimum/exporters/onnx/model_configs.py

```diff
@@ -1330,3 +1330,8 @@ def generate_dummy_inputs_for_validation(
             reference_model_inputs["encoder_hidden_states"] = reference_model_inputs.pop("encoder_outputs")[0]
 
         return super().generate_dummy_inputs_for_validation(reference_model_inputs)
+
+
+class FunnelTransformerOnnxConfig(BertOnnxConfig):
+    DEFAULT_ONNX_OPSET = 12
```

Comment:
looks like the issue when exporting the operator 'aten::einsum' was solved, the minimum required torch version can be fixed by setting `MIN_TORCH_VERSION`